### PR TITLE
[FINAL] Added line spacing support to MOAITextRenderer.

### PR DESCRIPTION
**Pull request by kstenerud**

### CODE REVIEW

File: src/moaicore/MOAITextRenderer.cpp [hunk 1]
```diff
@@ -64,11 +64,11 @@ int	MOAITextRenderer::_render ( lua_State *L ){
 	}
 	
 	cc8* text = state.GetValue < cc8* > (2, "");
-	float lineSpacing = 0.0f;
+	printf("LINE SPACING: %f\n", self->mLineSpacing);
```
Comment (JayAustria) [inline src/moaicore/MOAITextRenderer.cpp]: 1 survivor

**Comment by JayAustria** [comment]
Changes look good to me. Only saw a printf that could be removed and everything else looks correct. :shipit:

**Comment by aab29** [comment]
Looks pretty straightforward.  :shipit: after killing off that last surviving `printf()`.  :fax: :boom:  :gun:

**Comment by NachoSoto** [comment]
Looks good.

**Pull request merged**